perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -49,6 +49,76 @@
 )
 log = logging.getLogger()
 
+WORKSHEET_NAME_GCS = 'ls_metrics_gcsfuse'
+WORKSHEET_NAME_PD = 'ls_metrics_persistent_disk'
+
+
+def _count_number_of_files_and_folders(directory, files, folders):
+  """Count the number of files and folders in the given directory recursively.
+
+  Args:
+    directory: protobuf of the directory.
+    files: Number of files.
+    folders: Number of folders.
+
+  Returns:
+    Number of files and folders encountered till now.
+  """
+
+  files += directory.num_files
+  folders += directory.num_folders
+  for folder in directory.folders:
+    child_files = 0
+    child_folders = 0
+    child_files, child_folders = _count_number_of_files_and_folders(folder, child_files, child_folders)
+    files += child_files
+    folders += child_folders
+  return files, folders
+
+
+def _export_to_gsheet(folders, metrics, command, worksheet) -> None:
+  """Exports data to the Google Sheet.
+
+  Args:
+    folders: List containing protobufs of testing folders.
+    metrics: A dictionary containing all the result metrics for each testing folder.
+    command: Command to run the tests on.
+    worksheet: Google Sheet worksheet to upload the data.
+  """
+
+  # Changing directory to comply with "cred.json" path in "gsheet.py".
+  os.chdir('..')
+  gsheet_data = []
+  for testing_folder in folders:
+    num_files = 0
```

Comment:
Do you need these two initialization? Or just
num_files, num_folders = _count_number..(testing_folder, 0, 0) just work?